pkg/kvstore/allocator/allocator.go

```diff
@@ -143,7 +143,6 @@ func (k *kvstoreBackend) lockPath(ctx context.Context, key string) (*kvstore.Loc
 // DeleteAllKeys will delete all keys
 func (k *kvstoreBackend) DeleteAllKeys() {
 	kvstore.DeletePrefix(context.TODO(), k.basePrefix)
-	kvstore.DeletePrefix(context.TODO(), k.basePrefix)
```

Comment:
:thinking:

Reply:
This was added by me in a recent PR, so it does not affect any release branches ;)